src/DependencyInjection/TailwindExtension.php

```diff
@@ -53,9 +53,11 @@ public function getConfigTreeBuilder(): TreeBuilder
 
         $rootNode
             ->children()
-                ->scalarNode('input_css')
-                    ->info('Path to CSS file to process through Tailwind')
-                    ->defaultValue('%kernel.project_dir%/assets/styles/app.css')
+                ->arrayNode('input_css')
+                    ->prototype('scalar')->end()
+                    ->beforeNormalization()->castToArray()->end()
```

Comment:
Will the old configs work continue working? I mean, if people have scalar input_css value, this should still work for them? The bundle will just convert that string value to an array behind the scenes, so no BC breaks?

Reply:
Sure, that's why I added the `beforeNormalization` part.